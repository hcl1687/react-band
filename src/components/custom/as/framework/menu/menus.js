export default [{
  path: '/',
  name: 'home',
  icon: 'iconicon_Main-menu_My-Activities'
}, {
  path: '/test',
  name: 'test',
  icon: 'iconicon_Main-menu_My-Assignments'
}]
