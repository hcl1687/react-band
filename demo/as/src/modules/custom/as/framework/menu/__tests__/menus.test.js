import menus from '../menus'

describe('custom/as/framework/menu/menus', () => {
  it('should render correctly', () => {
    expect(menus).toMatchSnapshot()
  })
})
