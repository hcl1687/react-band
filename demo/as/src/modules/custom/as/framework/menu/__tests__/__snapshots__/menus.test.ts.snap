// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/framework/menu/menus should render correctly 1`] = `
Array [
  Object {
    "icon": "HomeOutlined",
    "name": "home",
    "path": "/",
  },
  Object {
    "icon": "AppstoreOutlined",
    "name": "assignment",
    "path": "/assignment",
  },
]
`;
